src/docs/forms/components/TextInput/examples/03-SimpleTextInput.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import Form, {
+    TextInput
+} from 'doj-react-adminlte/Form';
+
+export default class SimpleTextInput3 extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            name:"",
+            password:""
+        };
+    }
+
+    handleChange = (name, value) => {
+        this.setState({
+            [name]: value});
+    };
+
+    render() {
+        const {name,password} = this.state
+        return (
+            <div>
+                <Form onChange={this.handleChange}>
+                    <div className="row">
+                        <div className={"col-xs-12"}>
+                            <TextInput name="name" label="Username" value={name}
+                                       placeholder="Username  - 10 Max Length"
+                                       maxLength= {10}
+                                       feedbackIconClass="fa fa-user login-feedback"
+                                       feedbackIconLeft/>
+                        </div>
+                    </div>
+                    <div className="row">
+                        <div className={"col-xs-12"}>
```

Comment:
Minimize example space by turning this to `col-xs-6`. Also, please remove braces (`{}`) for string property values for consistency.